packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -171,3 +172,36 @@ def destroy_instance(self, instance: Instance) -> None:
+        """
+        try:
+            res = self.req.get(f'{self.baseurl}/v1/instances/{instance.id}')
+            data = res.json()
+            res.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise CorelliumApiException(data.get('error', str(e))) from e
+
+        for console in data.get('consoles', []):
+            if console['name'] == console_name:
+                return console['id']
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)